opengcs/Dockerfile

```diff
@@ -1,57 +0,0 @@
-# platform=linux
```

Comment:
Do we believe this is not needed anymore?

Reply:
I'd remembered hearing in passing this was obsolete now but should probably confirm that first

Reply:
This is at least the kind of case where it's really good to call out in the commit description "Removing Dockerfile as we believe it is no longer needed". Currently if we come back and look at this commit in the future we have NO idea why it was actually removed.

Reply:
Good call, the commit mentions deleting it but no explanation 😆